src/mmu/paging.c

```diff
@@ -187,11 +233,6 @@ void map_page_to_frame(uint64_t page, uint64_t frame, uint64_t flags)
 
   uint64_t frame_number = frame_containing_address(frame);
 
-  if (translate_page(page) == frame_number) {
```

Comment:
Removed this here because otherwise couldn't find the frame.

Reply:
yep, but that seems like a god check, no? (I remember having troubles with it when changing other mmu things)

Reply:
It says misaligned p1 page 262146 if I do this. Figured the problem tho, and restored this.